features/git-rename-branch/branch_does_not_exist.feature

```diff
@@ -0,0 +1,28 @@
+Feature: git rename-branch: errors if the feature branch does not exist
+
+  As a developer mistyping the feature branch name to rename
+  I should get an error that the given branch does not exist
+  So that I can rename the correct branch name.
```

Comment:
I think one doesn't rename names, one renames branches here. 
`So that I can rename the correct branch.`